### PR TITLE
Deleting CalculateResult methond and button in Admin View

### PR DESCRIPTION
**Pull request by Kamil4921**
## Screenshots (if appropriate)
image

## Types of changes

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/Index.cshtml [hunk 1]
```diff
@@ -16,7 +16,6 @@
             <h4>DateTimeUtc - @ViewBag.DateTimeUtc</h4>
             <div>
                 <a href="/Admin/AddTest" class="btn btn-default">@AddTest</a>
-                <a href="/Admin/CalculateResults" class="btn btn-default">@CalculateResults</a>
```
Comment (Kamil4921) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/Index.cshtml:19]: @WTobor Why after deleting this button it is still visible in application?

Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/Index.cshtml:19]: maybe cache? shift+F5 or clean all cookies etc

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_LoginPartial.cshtml [hunk 2]
```diff
@@ -18,12 +18,12 @@
 }
 else
 {
-    @*<ul>
+    <ul>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_LoginPartial.cshtml:21]: please do not add this code in PR

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/UnitTests/Controllers/AdminControllerTests.cs [hunk 3]
```diff
@@ -198,6 +198,7 @@ public void EndTest_TestIsStarted_ReturnsRedirectToAction()
             Assert.Equal("Index", redirectToActionResult.ActionName);
         }
 
+        /*
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/UnitTests/Controllers/AdminControllerTests.cs:201]: please do not comment out, just remove

**Comment by WTobor** [comment]
@Kamil4921 please resolve conflicts; is this PR ready? if yes, mark it as _Ready for review_ (for now it is a draft) :)

**Comment by WTobor** [comment]
@Kamil4921 please resolve conflicts

**Comment by WTobor** [comment]
@Kamil4921 what happened here in the history of commits? I can see changes, that are already on `develop`. Please merge with `develop` one more time and ensure that the history is clean and shows only your changes (commits)

**Pull request closed without merging**